public/app/features/expressions/components/Math.tsx

```diff
@@ -92,25 +89,34 @@ export const Math = ({ labelWidth, onChange, query, onRunQuery }: Props) => {
                       description="rounds the number down to the nearest integer value. It's able to operate on series or escalar values."
                     />
                   </div>
-                  <div>
-                    See our additional documentation on{' '}
-                    <a
-                      className={styles.documentationLink}
-                      target="_blank"
-                      href="https://grafana.com/docs/grafana/latest/panels/query-a-data-source/use-expressions-to-manipulate-data/about-expressions/#math"
-                      rel="noreferrer"
-                    >
-                      <Icon size="xs" name="external-link-alt" /> Math expressions
-                    </a>
-                    .
-                  </div>
                 </div>
               }
+              title={
+                <Stack gap={1} direction="row">
+                  <Icon name="book-open" /> Math operator
+                </Stack>
+              }
+              footer={
+                <div>
+                  See our additional documentation on{' '}
+                  <a
+                    className={styles.documentationLink}
+                    target="_blank"
+                    href="https://grafana.com/docs/grafana/latest/panels/query-a-data-source/use-expressions-to-manipulate-data/about-expressions/#math"
+                    rel="noreferrer"
+                  >
+                    <Icon size="xs" name="external-link-alt" /> Math expressions
+                  </a>
+                  .
+                </div>
+              }
+              closeButton={true}
+              placement="bottom-start"
             >
               <span>
                 Expression <Icon name="info-circle" />
```

Comment:
I think now that it doesn't show on hover, this content should have a pointer cursor so that the user knows it can click it, what do you think?

Reply:
💯

Reply:
done!